utils/swift_build_support/tests/mock-distcc.cmd

```diff
@@ -0,0 +1 @@
+@ python "%~dp0\mock-distcc" %*
```

Comment:
What happens if `python` is not in `PATH`?

Reply:
I tried a couple of alternatives here, but I couldn't find one that was great.

I tried injecting `sys.executable` from the Python side, but then the quoting do not work. Sending an environment variable only for this case also do not work.

A potential solution is trying to use `%PYTHON_HOME%\python.exe` or inject `sys.executable` as an environment variable from `lit.cfg`.

I can look into those solutions, but I would not mind simple, if it works in most cases (in our case, the CI setup).

Reply:
Hmm, `sys.executable` should usually work (although, with your favourite `%r` conversion)